### ISSUE TITLE
CSI300 components changes frequently, so how did you build the stock pool to generate the feature set?

### ISSUE THREAD

**Issue by rongzhou** [cq-dong/DFT_25#3]
In paper you use the stock correlation information in model, but the index components always changed.

**Comment by cq-dong** [comment]
> In paper you use the stock correlation information in model, but the index components always changed.在论文中，您在模型中使用股票相关性信息，但指数成分总是发生变化。
Yes, the stocks in CSI300 are not fixed, and there is no fixed relationship between the stocks in the input data and the index. Changes in stocks do not need to be considered because the model is not specific to a particular stock or combination
